src/isic-client.ts

```diff
@@ -5,10 +5,11 @@ export default class IsicClient extends OauthClient {
 
   constructor(
     clientId: string,
-    isicBaseUrl = 'https://api.isic-archive.com',
+    isicBaseUrl: string = 'https://api.isic-archive.com',
```

Comment:
FYI, TypeScript is smart enough to infer the type here, from the default value. There's nothing wrong with specifying it though.

Reply:
Or actually, the linting prefers you don't...